src/plugins/intel_cpu/tests/functional/single_layer_tests/group_convolution.cpp

```diff
@@ -484,6 +493,294 @@ INSTANTIATE_TEST_SUITE_P(smoke_GroupConv_3D_Gemm_BF16, GroupConvolutionLayerCPUT
                                 ::testing::Values(cpuBF16PluginConfig)),
                         GroupConvolutionLayerCPUTest::getTestCaseName);
 
+/* ============= GroupConvolution params (brgemm_1D) ============= */
+// disable kerenl size = 1
+// https://github.com/openvinotoolkit/oneDNN/blob/6df930dab5ab0a7dfaea6100acd03b479e2fa0a8/src/cpu/x64/jit_brgemm_conv_utils.cpp#L1833
+const std::vector<SizeVector> kernels_brgemm_1d = {{3}};
+const std::vector<SizeVector> strides_brgemm_1d = { {1}, {2} };
+const std::vector<std::vector<ptrdiff_t>> padBegins_brgemm_1d = { {0}, {1} };
+const std::vector<std::vector<ptrdiff_t>> padEnds_brgemm_1d = { {0} };
+const std::vector<SizeVector> dilations_brgemm_1d = { {1}, {2} };
+
+/* ============= GroupConvolution params (brgemm_2D) ============= */
+const std::vector<SizeVector> kernels_brgemm_2d = {{3, 3}};
+const std::vector<SizeVector> strides_brgemm_2d = {{1, 1}, {2, 2}};
+const std::vector<std::vector<ptrdiff_t>> padBegins_brgemm_2d = {{0, 0}, {1, 1}};
+const std::vector<std::vector<ptrdiff_t>> padEnds_brgemm_2d = {{0, 0}};
+const std::vector<SizeVector> dilations_brgemm_2d = {{1, 1}, {2, 2}};
+
+/* ============= GroupConvolution params (brgemm_3D) ============= */
+const std::vector<SizeVector> kernels_brgemm_3d = {{3, 3, 3}};
+const std::vector<SizeVector> strides_brgemm_3d = {{1, 1, 1}, {2, 2, 2}};
+const std::vector<std::vector<ptrdiff_t>> padBegins_brgemm_3d = {{0, 0, 0}, {1, 1, 1}};
+const std::vector<std::vector<ptrdiff_t>> padEnds_brgemm_3d = {{0, 0, 0}};
+const std::vector<SizeVector> dilations_brgemm_3d = {{1, 1, 1}, {2, 2, 2}};
+/* ============= */
+
+// disable small channel <= 16
+// https://github.com/openvinotoolkit/oneDNN/blob/6df930dab5ab0a7dfaea6100acd03b479e2fa0a8/src/cpu/x64/jit_brgemm_conv_utils.cpp#L1712
+const SizeVector numGroups_brgemm_Blocked = {2};
+
+/* ============= GroupConvolution (brgemm 1D) ============= */
+const auto groupConvParams_ExplicitPadding_brgemm_1D = ::testing::Combine(
+        ::testing::ValuesIn(kernels_brgemm_1d),
+        ::testing::ValuesIn(strides_brgemm_1d),
+        ::testing::ValuesIn(padBegins_brgemm_1d),
+        ::testing::ValuesIn(padEnds_brgemm_1d),
+        ::testing::ValuesIn(dilations_brgemm_1d),
+        ::testing::ValuesIn(numOutChannels_Blocked),
+        ::testing::ValuesIn(numGroups_brgemm_Blocked),
+        ::testing::Values(ngraph::op::PadType::EXPLICIT)
+);
+
+// disable small shape on amx
+//  https://github.com/openvinotoolkit/oneDNN/blob/6df930dab5ab0a7dfaea6100acd03b479e2fa0a8/src/cpu/x64/jit_brgemm_conv_utils.cpp#L1719
```

Comment:
Good to have the reference, but I think it is better to have it on the ticket instead.
Here we could just add a justification. for example:
"oneDNN does not support brgemm for" and list the conditions.
Also is it possible to have few test instances for brgemm with unsupported parameters and expect it to fail? Thus the limits will be explicitly tested and, for example, if oneDNN will change the brgemm restrictions we will notice it.

Reply:
And name them explicitly, for example:
``` cpp

const Shape smallNumOfInputChannels{x,x,x};
std::vector<InputShape> unsupported_brgemm_1d = {
    {{}, {smallNumOfInputChannels}
};
```
And expect it to fail.

Reply:
@luo-cheng2021 @EgorDuplensky  I add brgemm 1x1 and some test case expected to be failed